lib/board.rb

```diff
@@ -13,4 +13,50 @@ def place_mark(mark, position)
     pos = position.to_i - 1
     @cells[pos] = mark
```

Comment:
Nitpick: can you rename `pos` to something else? The current name doesn't really indicate what it is (which is more of a mapping from a user's choice to a specific board implementation)

Reply:
ok, done!